packages/moonstone/DatePicker/DatePicker.js

```diff
@@ -63,7 +66,27 @@ const dateTimeConfig = {
 			.match(/([mdy]+)/ig)
 			.map(s => s[0].toLowerCase());
 
-		return {formatter, order};
+		/**
+		 * Converts a gregorian year to local year
+		 *
+		 * @param	{string}	year	gregorian year string
```

Comment:
Are these really strings?  The prop is a number.

Reply:
done!